crates/libtiny_tui/src/messaging.rs

```diff
@@ -135,56 +135,48 @@ impl MessagingUI {
         self.msg_area.draw(tb, colors, pos_x, pos_y);
     }
 
-    pub(crate) fn keypressed(&mut self, key: Key) -> WidgetRet {
-        match key {
-            Key::Ctrl('c') => {
+    pub(crate) fn keypressed(&mut self, key_action: Option<KeyAction>) -> WidgetRet {
```

Comment:
This function does not handle `None` case (returns `KeyIgnored`), I think if we make the argument `KeyAction` instead of `Option<KeyAction>` it becomes clear to the caller that this does not handle when a key is not mapped to an action.

If we do that then we decide what to do when a key is not mapped in the code where we try to map the key to an action. It's much nicer, I don't need to follow the value in the code base to find out where the `None` case is handled and how.

Could you implement that? Should be a few lines of change in `TUI::keypressed`.

Reply:
Done!